runtime/ftplugin/context.vim

```diff
@@ -108,6 +104,11 @@ if get(g:, 'context_mappings', 1)
     endif
 endif
 
+if !exists('current_compiler')
+  b:undo_ftplugin ..= "| compiler make"
+  compiler context
+endif
+b:undo_ftplugin ..= "| sil! delc -buffer ConTeXt | sil! delc -buffer ConTeXtLog | sil! delc -buffer ConTeXtJobStatus | sil! delc -buffer ConTeXtStopJobs"
```

Comment:
```suggestion
  let b:undo_ftplugin ..= "| compiler make"
  compiler context
endif

let b:undo_ftplugin ..= "| sil! delc -buffer ConTeXt | sil! delc -buffer ConTeXtLog | sil! delc -buffer ConTeXtJobStatus | sil! delc -buffer ConTeXtStopJobs"
```

Reply:
Sorry, forgot that this was vim9script. Should I drop the change entirely? I think that would be more consistent.

Reply:
If you drop the change than it's inconsistent with other runtime files :)